config/locales/de.yml

```diff
@@ -152,7 +152,8 @@ de:
       create: &id-volunteers-create
         <<: *id-availability
         acceptance: *id-volunteer-acceptance
-        availability: Wann sind Sie für einen Freiwilligeneinsatz verfügbar?
+        availability: Verfügbarkeit?
+        availability_question: Wann sind Sie für einen Freiwilligeneinsatz verfügbar?
```

Comment:
Since this translation was introduces here, should there be a call for this in the form in this PR?

Reply:
The translation was missing, so it hat to be put in place.

Reply:
For some reason I didn't think of that case :P